.travis.yml

```diff
@@ -9,7 +9,12 @@ node_js:
   - '8'
   - '10'
   - '12'
-install: npm install
+install:
+  - npm install
+  # as requested by the React team :)
+  # https://reactjs.org/blog/2019/10/22/react-release-channels.html#using-the-next-channel-for-integration-testing
+  - if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then npm install react@next
+    react-dom@next; fi
```

Comment:
Whoops! Somehow this ended up on two lines 🙃 Maybe it'll work anyway, but I'll fix that. Should be on one line.

Reply:
You can manually add a `\` at the end of line to signify multi-line statements.

Going to assume your prettier messed it up.

Reply:
Prettier doesn't typically introduce bugs. I just ran it locally and checked and prettier didn't touch this line. I'm pretty sure this happened because I just used vim and probably hit a key by mistake.